Makefile

```diff
@@ -1,7 +1,7 @@
 SHELL := /bin/bash
 export GO111MODULE ?= on
-export VERSION ?= v1.0.0
-export ECE_VERSION ?= 2.6.0
+export VERSION ?= v1.1.0
+export ECE_VERSION ?= 2.7.0
```

Comment:
This hasn't been released yet though. Also I'm pretty sure there's going to be a 2.6.1 release with some fixes 🤔 

I was also pretty sure we only updated this env var when we updated the api spec, which we haven't done?

Reply:
> This hasn't been released yet though. Also I'm pretty sure there's going to be a 2.6.1 release with some fixes 🤔

Yep, that's why I created a `1.0` branch and master can now look forward to the next feature version.

> I was also pretty sure we only updated this env var when we updated the api spec, which we haven't done?

`ECE_VERSION` should be updated to track the version which the `cloud-sdk-go` is supposed to target, since we're looking at releasing `1.1.0` with ECE `2.7.0` updating those two together makes sense, since the next time we update the spec, it will be the 2.7.0 spec.

Does that make sense?

Reply:
Before we change the SDK to use branches instead of tags for our releases, I think it's probably a good idea to talk to the rest of the team, since the main cloud repo is no longer using branches but tags for releases 🤔 I'm not particularly for or against tags or branches, but I really do think it makes sense to have some consistency between repos. What do you think?

Reply:
checked the cloud repo again and realised they're not doing what I thought they were doing 😄 nvm this comment